File: app/src/androidTest/java/com/latnok/atmapp/Pinvalidatortest.java
package com.latnok.atmapp;

import org.junit.Test;
import static  org.junit.Assert.assertFalse;
import static  org.junit.Assert.assertTrue;

public class Pinvalidatortest {

    @Test
    public void pinvalidator_correctpin_returnstrue(){
        assertFalse(Transaction.isvalid("2222"));
    }


}

File: app/src/androidTest/java/com/latnok/atmapp/Pinvalidatortest.kt
package com.latnok.atmapp

import android.app.Activity

import org.junit.Test
import org.junit.Assert.assertFalse
import org.junit.Assert.assertThat
import org.junit.Assert.assertTrue

class Pinvalidatortest {

    @Test
    fun pinvalidator_correctpin_returnstrue() {
        assertFalse(MainActivity.isvalid("2222"))
        assertFalse(Transaction.isvalid("2222"))
    }

    @Test
    fun pinvalidator_wrongpin_returnstrue() {
        assertFalse(MainActivity.isvalid(""))
        assertFalse(Transaction.isvalid(""))
    }

    @Test
    fun pinvalidator_null_returnstrue() {
        assertFalse(MainActivity.isvalid(null))
        assertFalse(Transaction.isvalid(null))
    }

}
